src/consensus/validation.h

```diff
@@ -135,12 +135,6 @@ class ValidationState
 
         return m_reject_reason;
     }
-    bool Valid()
-    {
-        m_result = Result{};
-        m_mode = ModeState::M_VALID;
-        return true;
-    }
```

Comment:
In 6f92462634b7856579b2014cc7cbc13c51657304:

In future can you edit/replace commits rather than changing the same code in multiple commits? It makes reviewing much easier.

Reply:
sorry about that, I thought it was better to add commits instead of force pushing 

will rebase in future